util/nextFonts.ts

```diff
@@ -0,0 +1,15 @@
+import { Source_Sans_Pro, Open_Sans } from 'next/font/google';
```

Comment:
We should add some information about usage
```suggestion
// source sans pro it's font for old design
// open sans it's for website redesign
import { Source_Sans_Pro, Open_Sans } from 'next/font/google';
```

Reply:
Ill add another document about this later!